.gitignore

```diff
@@ -101,3 +101,5 @@ version.ml
 version.texi
 .cargo
 !src/vendor/**
+**/target/rls/debug/
+/src/target/
```

Comment:
I could have sworn I've seen `src/target` show up without ever running `cargo` in there. But I just realized that the "Rust Enhanced" Sublime Text plugin runs cargo when saving. So you're probably right that it's not related to the build system.